src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1377,7 +1377,8 @@ void RaftPart::processAppendLogRequest(
     lastMsgRecvDur_.reset();
 
     if (req.get_sending_snapshot() && status_ != Status::WAITING_SNAPSHOT) {
-        LOG(INFO) << idStr_ << "Begin to wait for the snapshot";
+        LOG(INFO) << idStr_ << "Begin to wait for the snapshot"
```

Comment:
"Begin to wait for the snapshot " ?

Reply:
Yeah, if leader found its peer need to get snapshot, it will send a request with `sending_snapshot = true`, and follower will change its own status to `WAIT_SNAPSHOT`.